communities/asu-phoenix/community-assessment.md

```diff
@@ -0,0 +1,121 @@
+        - Fulton Ambassadors 
+        - Fulton Student Council 
+        - Polytechnic Campus Orgs
+        - [Fulton Student Orgs](https://fso.engineering.asu.edu/student-org-directory/)
+
+- Phoenix Area, many partnerships + meetups 
+    - Galvanize, great co-working/event space
+    - #YesPHX Movement (Tech Startup Movement)
+    - Meetups 
+        - Code For America
+        - VR/AR Hangouts 
+        - IoT Meetups 
+    - Hackerrnest socials 
+    - Local Companies 
+        - GoDaddy 
+        - Intel 
+        - Amazon
+        - American Express 
+
+### Accessibility
+- There is no central area where all the resources from each club is placed
+    - Most are put on SunDevilSync or a link provided during the event
+- Club Meetings usually happen Monday-Thursday in between 6-8pm
+
+### Interests
+
+- Mostly, but we would like to welcome anyone interested in technology
+- There are several other majors that have a tech influence that we would love to get involved
+    - Digital Culture (Art)
+    - Technical Management (Polytechnic Campus)
+    - Graphic Information Systems (Polytechnic Campus)
+    - Computer Systems Engineering
+    - Computing Informatics Systems (Business)
+
+### Goals
+
+- Most people get involved to make connections, find jobs/internships, learn new skills
+- We would like to provide a larger scale event to really get everyone who has these goals all in one. Furthermore, have them develop connections with the existing communties and encourage them to stay invovled. 
+
+
+### Location
+
+- **ASU Tempe/Poly Campus**
+    - Variety of classrooms and lecture halls across campus, Free to Use 
+    - Usually in person meetings
+    - Not much free parking around ASU unless it is the weekend
+    - Transportation: 
+        - Light Rail 
+        - Intercampus Shuttles 
+        - Orbit Bus 
+        - Regular Bus 
+    - Can fit up to 30-400 in a classroom, chair w/small desk setting 
+    - Can use for presentation  
+- **Galvanize**
+     - Public space - Free to Use / Other space needs to be reserved
+    - Usually in person meetings
+    - Free Parking 
+    - Transportation: 
+        - Light Rail 
+        - Uber
+    - Can fit up to 100 in a tier seating, 325 in amphitheatre, 500 in total, spread out
+- **CO+HOOTS**
+    - Need to find more info, but have heard it is a good coworking place 
+
+### Communication
+- How does your community get in touch with one another?
+    - Facebook, Email Lists, School Postings
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+    - Not at this moment, only for the individual organizations on campus
+    - Computer Science Facebook group, but not many active users
+- Could your organization benefit from one or more of those, or another way of communicating?
+    - Definitely, especially one that students will actually actively use for discussion, rather than just for class guides/other less relevant activity
```

Comment:
>  especially one that students will actually actively use for discussion
This is always the tricky bit! Do you have an idea of why the current Facebook groups may not be providing this?